pkg/porter/cnab.go

```diff
@@ -239,6 +239,10 @@ func (o *bundleFileOptions) validateCNABFile(cxt *context.Context) error {
 // LoadParameters validates and resolves the parameters and sets. It must be
 // called after porter has loaded the bundle definition.
 func (o *sharedOptions) LoadParameters(p *Porter) error {
+	if o.combinedParameters != nil {
```

Comment:
Can we add a comment here around this logic?  Do we return immediately because o.combinedParameters being non-nil means they've been processed sufficiently?

Reply:
Yup, it's because I noticed that we were re-processing the parameters, so this detects if we've already done it. I'll add a comment.